tests/testsuite/build_script_env.rs

```diff
@@ -48,12 +48,63 @@ fn rerun_if_env_changes_config() {
 
     p.cargo("check")
         .with_status(101)
-        .with_stderr_data(
-            "\
+        .with_stderr_data(str![[r#"
 [COMPILING] foo v0.1.0 ([ROOT]/foo)
 [ERROR] failed to run custom build command for `foo v0.1.0 ([ROOT]/foo)`
-...",
+...
+"#]])
+        .run();
+}
+
+#[cfg(windows)]
+#[cargo_test]
+fn rerun_if_env_changes_config_in_windows() {
```

Comment:
Can you please add all tests in commits before any functionality changes?

Reply:
If I understand correctly, they should all be added.